src/lyapunovspectrum.jl

```diff
@@ -157,18 +182,22 @@ function liap_taylorstep!(f, t::Taylor1{T}, x::Vector{Taylor1{U}}, dx::Vector{Ta
 end
 
 """
-    liap_taylorinteg(f, q0, t0, tmax, order, abstol; maxsteps::Int=500)
+    lyap_taylorinteg(f!, q0, t0, tmax, order, abstol[, f!]; maxsteps::Int=500)
```

Comment:
Probable some sort of typo here.

Reply:
Thanks for noticing this! the `[...]` were intended to denote an optional parameter, but perhaps I'm using the wrong convention for optional arguments in the docstrings? Anyway, this is still WIP so the final signature call may change

Reply:
Regarding this, I was simply confused; perhaps, it was the "," in between: `[, ...]`, instead of `, [...]`.